api/v3/openapi.yaml

```diff
@@ -136,7 +163,7 @@ paths:
           $ref: '#/components/responses/ServerErrorResponse'
 
     post:
-      summary: 'Create ACLs'
+      summary: 'Create an ACL'
       operationId: createKafkaV3Acls
```

Comment:
nit: remove `s` at the end?

Reply:
I don't know if this will have an unexpected effect downstream.  @linouk23 might know?  If not, I'm leaving the `s` on the end to avoid breaking eg the docs or the go sdk generation.